src/pytorch_lightning/strategies/utils.py

```diff
@@ -13,6 +13,20 @@
 # limitations under the License.
 import os
 
+import torch
+
+from pytorch_lightning.utilities.enums import PrecisionType
+
 
 def on_colab_kaggle() -> bool:
```

Comment:
Oh dang, we wanted to remove this and the file with it. 
Guess we forgot to deprecate it :( Will do it.